msg/tools/uorb_rtps_message_ids.yaml

```
@@ -325,4 +325,10 @@ rtps:
   - msg: vehicle_visual_odometry_aligned
     id: 169
     alias: vehicle_odometry
+    msg: ekf2_innovation_variances
```

you are missing the dash here:
```suggestion
  - msg: ekf2_innovation_variances
```